Task 2/Task 2.1/Task 2.1.2/Polygons/Polygon.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Linq;
+
+namespace Task_2._1._2.Polygons
+{
+    public abstract class Polygon : Figure
```

Comment:
Polygon is usually a figure with more than 2 sides, Line - isn't really a polygon.. But it kinda is in maths..